hydra/core/override_parser/overrides_visitor.py

```diff
@@ -76,7 +76,7 @@ def is_ws(self, c: Any) -> bool:
         return isinstance(c, TerminalNodeImpl) and c.symbol.type == OverrideLexer.WS
 
     def visitPrimitive(
-        self, ctx: OverrideParser.PrimitiveContext
+        self, ctx: Union[OverrideParser.PrimitiveContext, OverrideParser.DictKeyContext]
```

Comment:
This is extending generated code, right?
I suspect you edited it by hand to get rid of the mypy type error.
If so, please refactor to call a common function with union signature.

Reply:
Just to be sure it's clear, this class itself isn't generated: it is a subclass of a generated class. So nothing bad happens when we regenerate the code for instance.

I had considered creating a common function but thought it was a bit overkill... anyway, I did it in ee5eaa807 :)

Reply:
dunno why I keep missing comments in fucking GitHub.
Yes, it's an extension of a generated class.
changing the variable types in a subclass is a code smell that will likely get flagged by automated tool as an error (if not now than once they pick up on my ultimate wisdom :) ).